tests/sentry/templatetags/test_sentry_api.py

```diff
@@ -18,4 +18,4 @@ def test_escapes_js(self):
         result = self.TEMPLATE.render(context={"org": org})
 
         assert "<script>" not in result
-        assert "\u003cscript\u003ealert(1);\u003c/script\u003e" in result
+        assert "\\u003cscript\\u003ealert(1);\\u003c/script\\u003e" in result
```

Comment:
`\u` only has semantics in unicode strings, and depending on Python version this is a different type of string.